src/EditorFeatures/CSharpTest/Diagnostics/SimplifyTypeNames/SimplifyTypeNamesTests.cs

```diff
@@ -3755,7 +3755,6 @@ void M()
     }
 }",
                 options: Option(CodeStyleOptions.QualifyPropertyAccess, false, NotificationOption.Warning),
-                diagnosticCount: 1,
```

Comment:
And in several other tests.

---
In reply to: [123327253](https://github.com/dotnet/roslyn/pull/18718#discussion_r123327253) [](ancestors = 123327253)